packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -104,7 +91,8 @@ export class DocumentStorageService implements IDocumentStorageService {
     }
 
     public async readBlob(blobId: string): Promise<ArrayBufferLike> {
-        const iso = IsoBuffer.from(await this.read(blobId), "base64");
+        const blob = await this.readBlob(blobId);
+        const iso = IsoBuffer.from(blobToString(blob), "utf8");
```

Comment:
I do not understand what this line does and it looks wrong. Why utf8? We have no clue what's the content of blob, it can be binary.